func.go

```diff
@@ -50,9 +50,9 @@ func RegisterLibFunc(fptr interface{}, handle uintptr, name string) {
 //	int16 <=> int16_t
 //	int32 <=> int32_t
 //	int64 <=> int64_t
-//	float32 <=> float (WIP)
-//	float64 <=> double (WIP)
-//	struct <=> struct (WIP)
+//	float32 <=> float
+//	float64 <=> double
```

Comment:
Can we split this PR into supporting floats and supporitng structs?

Reply:
Floats are already supported in purego. I just changed the comment in this PR. Do you want the comment change in a separate PR? This PR only adds support for struct arguments not returns

Reply:
> Do you want the comment change in a separate PR?

Yes. This would be less confusing.